bigtable/tests/unit/test_batcher.py

```diff
@@ -0,0 +1,200 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+import mock
+
+from ._testing import _make_credentials
+
+
+class TestMutationsBatcher(unittest.TestCase):
+    from grpc import StatusCode
+
+    PROJECT_ID = 'project-id'
+    INSTANCE_ID = 'instance-id'
+    INSTANCE_NAME = ('projects/' + PROJECT_ID + '/instances/' + INSTANCE_ID)
+    TABLE_ID = 'table-id'
+    TABLE_NAME = INSTANCE_NAME + '/tables/' + TABLE_ID
+
+    # RPC Status Codes
+    SUCCESS = StatusCode.OK.value[0]
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigtable.table import Table
+
+        return Table
+
+    def _make_table(self, *args, **kwargs):
+        return self._get_target_class()(*args, **kwargs)
+
+    @staticmethod
+    def _get_target_client_class():
+        from google.cloud.bigtable.client import Client
+
+        return Client
+
+    def _make_client(self, *args, **kwargs):
+        return self._get_target_client_class()(*args, **kwargs)
+
+    def test_constructor(self):
+        from google.cloud.bigtable.batcher import MutationsBatcher
+
+        credentials = _make_credentials()
+        client = self._make_client(project='project-id',
+                                   credentials=credentials, admin=True)
+
+        instance = client.instance(instance_id=self.INSTANCE_ID)
+        table = self._make_table(self.TABLE_ID, instance)
+
+        mutation_batcher = MutationsBatcher(table)
+        self.assertEqual(table.table_id, mutation_batcher.table.table_id)
+
+    def test_add_row(self):
+        from google.cloud.bigtable.batcher import MutationsBatcher
+        from google.cloud.bigtable.row import DirectRow
+
+        table = _Table(self.TABLE_NAME)
```

Comment:
I recommend that you create methods for the common code for each test to reduce the repetitive code.